cmd/minishift/cmd/start.go

```diff
@@ -134,9 +143,14 @@ func runStart(cmd *cobra.Command, args []string) {
 	// we need to determine whether this is a restart prior to potentially creating a new VM
 	isRestart := cmdutil.VMExists(libMachineClient, constants.MachineName)
 
-	hostVm, ip := startHost(libMachineClient)
+	fmt.Printf("-- Starting local OpenShift cluster")
+	hostVm := startHost(libMachineClient)
 	registerHost(libMachineClient)
 
+	// preflight checks (after start)
+	preflightChecksAfterStartingHost(hostVm.Driver)
```

Comment:
I think these tests are making sure OpenShift works fine after `clusterup`. So why not changing it to preflightChecksBeforeClusterUp()?